src/Apps/Sell/Routes/AdditionalRoutes/ConditionRoute.tsx

```diff
@@ -52,16 +53,24 @@ interface ConditionRouteProps {
 }
 
 export const ConditionRoute: React.FC<ConditionRouteProps> = props => {
+  const { actions } = useSellFlowContext()
   const submission = useFragment(FRAGMENT, props.submission)
   const artwork = submission.myCollectionArtwork
-  const { actions } = useSellFlowContext()
 
   const [
     isConditionDefinitionModalOpen,
     setIsConditionDefinitionModalOpen,
   ] = useState(false)
 
-  if (!artwork) return null
+  if (!artwork) {
+    return (
+      <ErrorPage
+        code="Something went wrong."
+        message="The artwork could not be loaded. Please try again or contact support@artsy.net."
+        m={4}
+      />
+    )
+  }
```

Comment:
Displaying a proper error page in case the My Collection artwork isn't present or cannot be loaded.

Reply:
We probably need to ensure that the associated MyC artwork cannot be deleted when it has an associated submission. If the artwork no longer exists, it is not possible to access this step and update the condition .

Reply:
If the artwork that has submission is removed from MyCollection, submission will still exist, right? What happens to the submission in this case? 
Probably an edge-case, asking mostly out of curiosity in case we talked about this before

Reply:
That's a good question! I will investigate 🔍  We should probably not allow removing the MyC artwork when it has a submission that is not in Draft state.

Reply:
https://artsyproduct.atlassian.net/browse/ONYX-1156

Reply:
cc @MounirDhahri